main.go

```diff
@@ -582,20 +582,46 @@ func main() {
 		},
 		{
```

Comment:
We should remove this subcommand and just set `Aliases: []string{"totp", "htop"}` on the `otp` subcommand.